codegen/src/test/java/com/paypal/selion/plugins/DataReaderTest.java

```diff
@@ -50,21 +52,28 @@ public void getKeys_v1() throws Exception {
 
     @Test
     public void getKeys_v2() throws Exception {
-        DataReader r = new DataReader("src/test/resources/SampleV2YamlPage.yaml");
+        DataReader r = new DataReader(YAML_V2);
         List<String> keys = r.getKeys();
         assertTrue(keys.contains("requestAPICredentialsLink"));
     }
 
     @Test
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/SeLion/paypal-SeLion/file/2360069956/issues/source?bid=2928425&fileBranchId=3219780#l61)